content/tutorial/tutorial.md

```diff
@@ -106,9 +106,10 @@ cd ..
 
 ```js
 import React from 'react';
-import ReactDOM from 'react-dom';
+import ReactDOM from 'react-dom/client';
 import './index.css';
 ```
+Copy the JS and CSS from the **[Starter Code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)** and add it in your local in `index.js` and `index.css` respectively.
```

Comment:
I believe we already have this documented right? 
<img width="560" alt="Screenshot 2022-05-05 at 8 50 40 PM" src="https://user-images.githubusercontent.com/32865581/166956783-e12ea784-f9fc-4e80-b978-748c2b94bd72.png">

Reply:
Missed to check the highlighted text. Removing those